src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -117,10 +120,29 @@ public static void addDocumentToBundle(FooterService footerService, SscsCaseData
     }
 
     public static DocumentType getPostHearingReviewDocumentType(PostHearing postHearing) {
-        if (SetAsideActions.REFUSE.equals(postHearing.getSetAside().getAction())) {
-            return DocumentType.SET_ASIDE_REFUSED;
-        } else if (CorrectionActions.REFUSE.equals(postHearing.getCorrection().getAction())) {
-            return DocumentType.CORRECTION_REFUSED;
+        if (nonNull(postHearing.getReviewType())) {
+            switch (postHearing.getReviewType()) {
+                case SET_ASIDE:
+                    if (SetAsideActions.REFUSE.equals(postHearing.getSetAside().getAction())) {
+                        return DocumentType.SET_ASIDE_REFUSED;
+                    }
+                    break;
+                case CORRECTION:
+                    if (CorrectionActions.REFUSE.equals(postHearing.getCorrection().getAction())) {
+                        return DocumentType.CORRECTION_REFUSED;
+                    }
+                    break;
+                case STATEMENT_OF_REASONS:
+                    if (StatementOfReasonsActions.REFUSE.equals(postHearing.getStatementOfReasons().getAction())) {
+                        return DocumentType.STATEMENT_OF_REASONS_REFUSED;
+                    }
+
+                    return DocumentType.STATEMENT_OF_REASONS_GRANTED;
+                case LIBERTY_TO_APPLY:
+                case PERMISSION_TO_APPEAL:
```

Comment:
Please throw an exception here to the effect of "review type not yet implemented" as a safeguard for us when we do implement those, that they need to be added here.

Reply:
done